### PR TITLE
chore: add vaults v1.5 fee support

### PR DESCRIPTION
**Pull request by axejintao**
# Summary

- add support from sdk for v1.5 fees + aum fee addition
- fix chain query param on debank api

### CODE REVIEW

File: src/rewards/rewards.controller.ts [hunk 1]
```diff
@@ -23,11 +23,11 @@ export class RewardsController {
   @Description('Returns a paginated chunk of reward balance snapshots for users')
   @Returns(200)
   async list(
-    @QueryParams('chain_id') chainId: string,
+    @QueryParams('chain_id') chainId?: Network,
     @QueryParams('page_num') pageNum?: number,
     @QueryParams('page_count') pageCount?: number,
   ): Promise<ListRewardsResponse> {
-    const chain = Chain.getChainById(chainId);
+    const chain = Chain.getChain(chainId);
```
Comment (btcookies) [inline src/rewards/rewards.controller.ts:30]: is this right? looked like func above was modifying getChainById

**Review by btcookies — DISMISSED** [review]
LGTM

File: src/rewards/rewards.controller.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { formatBalance } from '@badger-dao/sdk';
+import { formatBalance, Network } from '@badger-dao/sdk';
```
Comment (kingikalgo) [inline src/rewards/rewards.controller.ts:1]: I think we can remove this

**Review by kingikalgo — APPROVED** [review]

**Review by btcookies — APPROVED** [review]
LGTM

**Pull request merged**